src/doc/guide.md

```diff
@@ -4231,7 +4231,7 @@ arguments, really powerful things are possible.
 
 Let's make a closure:
 
-```{rust}
+```{rust,ignore}
```

Comment:
I'd be great if this landed with updated docs, so I'll go with the first option.